scoreboard/attachments/__init__.py

```diff
@@ -40,11 +41,23 @@ def get_backend_type():
     backend = get_backend_path()
     return urlparse.urlparse(backend).scheme
 
+def patch(_backend_type):
+    if _backend_type == "file":
+        import file as backend
+    elif _backend_type == "gcs":
+        import gcs as backend
+    elif _backend_type == "test":
+        import testing as backend
+    else:
+        raise ImportError('Unhandled attachment backend %s' % _backend_type)
+    globals()['backend'] = backend
 
 _backend_type = get_backend_type()
 if _backend_type == "file":
-    from scoreboard.attachments.file import *
+    import file as backend
```

Comment:
Is there a reason the import style had to change?

Specifically, I get you wanted to import the module as backend, but `from scoreboard.attachments import file as backend` achieves that.  

Python 3 has actually removed this implicit relative import, so I'd like to keep the full package paths where possible.  (Yes, we need to fix a lot of other things for py3k support, but I'd rather not break new things.)